src/main/java/wooteco/subway/auth/ui/AuthenticationPrincipalArgumentResolver.java

```diff
@@ -24,6 +27,7 @@ public boolean supportsParameter(MethodParameter parameter) {
     @Override
     public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
```

Comment:
```suggestion
    public Member resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer,
        NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
```

도메인 `Member` 객체를 이용하고 있는데, 이렇게 구현하신 이유가 있을까요? 도메인을 직접 이용하면 어떤 문제가 발생할 수 있을까요?

Reply:
DTO가 있다는 사실을 깜빡했네요. 😅
아무래도 뷰나 컨트롤러에서  도메인 기능을 조작할 수 있게 되서 원치 않는 결과를 초래할 수 있겠네요.

Reply:
Member가 아니라 전용 DTO를 만들어야겠군요.
아무래도 뷰나 컨트롤러에서 도메인 기능을 조작할 수 있게 되서 원치 않는 결과를 초래할 수 있겠네요.